orders/test/interface/test_service.py

```diff
@@ -0,0 +1,74 @@
+import pytest
```

Comment:
Might be worth a module-level docstring here to define the scope of an "interface" test for a nameko service